README.md

```diff
@@ -1,23 +1,38 @@
-# streamlink-portable
-A portable version builder of Streamlink for Windows
-<br>
-<br>
-<B>Instructions (End User):</B>
-<br>
--Download https://github.com/streamlink/streamlink-portable/archive/master.zip
-<br>
--Extract the "Streamlink for Windows (Compiled)" folder from the ZIP file
-<br>
--Check README.txt file for requirements and run "Streamlink for Windows.exe"
-<br>
-<br>
-<B>Instructions (Developer):</B>
-<br>
--Download https://github.com/streamlink/streamlink-portable/archive/master.zip
+# Download:
+
+You can get the latest git archive <a href="https://github.com/streamlink/streamlink-portable/archive/master.zip">here</a>.
 <br>
--Extract both folders from the ZIP file
+Inside the ZIP you will find two folders:
+* Streamlink for Windows (Compiled): The program executable, intended for end users.
+* Streamlink for Windows (Source code): The program source code, intended for developers.
+
+# FAQs:
+
+### What is Streamlink Portable?
+A program to build a portable version of Streamlink for Windows.
+
+### What are the supported platforms?
+Windows Vista SP2 or later with .NET Framework 4.5+.
+
+### What can i do with it?
+Build between three flavors:
+* Portable EXE (Recommended): For users who prefer to have all available files in the same folder.
+* Standalone EXE: For users who prefer to have all files packed within the executable.
+* Portable BAT: Same as Portable EXE but a batch file is generated instead of an executable.
+
+By default the building source is the <a href="https://github.com/streamlink/streamlink/commits/master">latest available commit</a> (nightly).
 <br>
--Open the Visual Studio Project inside the "Streamlink for Windows (Source code)" folder and compile it
+No system files are modified and you can run it from anywhere (such as flash or external drives).
+
+### Can i download a ready-to-use release without using the program?
+Yes, you can find precompiled releases <a href="https://github.com/streamlink/streamlink-portable/releases">here</a>.
 <br>
--Copy the "Streamlink for Windows (Compiled)\Files" folder to "Streamlink for Windows (Source code)\bin\Debug" if needed
+Note that only stable releases are available, for nightly you have to use the builder.
```

Comment:
`for nightly you have to use the builder.` to `for nightly builds you have to use the program.` the term `builder` is only used at one other point on line 2 and might be confusing as it is referred to as a program on line 27.